sunset/code/modules/mob/living/carbon/human/species.dm

```diff
@@ -10,6 +10,18 @@
 			var/obj/item/bodypart/B = O
 			B.should_draw_sunset = FALSE
 
+/datum/species
+	// H for human because vox was setup like this already
+	var/sound/Hfemale_scream_sound = 'sound/effects/mob_effects/goonstation/female_scream.ogg'
```

Comment:
Why prefixed with an H?

Reply:
I will answer the others later, but vox was setup already with the un-prefixed var names. So instead I just added the H for human

Reply:
I updated the comment a little more to try and make better sense.